docs/cli.md

```diff
@@ -48,6 +48,91 @@ Either way, running `clai` will start an interactive session where you can chat
 - `/multiline`: Toggle multiline input mode (use Ctrl+D to submit)
 - `/cp`: Copy the last response to clipboard
 
+### Web Chat UI
+
+Launch a web-based chat interface for your agent:
+
+```bash
+clai --web --agent module:agent_variable
+```
+
+For example, if you have an agent defined in `my_agent.py`:
+
+```python
+from pydantic_ai import Agent
+
+my_agent = Agent('openai:gpt-5', system_prompt='You are a helpful assistant.')
+```
+
+Launch the web UI with:
+
+```bash
+clai --web --agent my_agent:my_agent
+```
+
+This will start a web server (default: http://127.0.0.1:8000) with a chat interface for your agent.
+
+#### Web Command Options
+
+- `--host`: Host to bind the server to (default: 127.0.0.1)
+- `--port`: Port to bind the server to (default: 8000)
+- `--config`: Path to custom `agent_options.py` config file
+- `--no-auto-config`: Disable auto-discovery of `agent_options.py` in current directory
+
+#### Configuring Models and Tools
+
+You can customize which AI models and builtin tools are available in the web UI by creating an `agent_options.py` file:
+
+```python title="agent_options.py"
+from pydantic_ai.builtin_tools import CodeExecutionTool, WebSearchTool
+from pydantic_ai.ui.web import AIModel, BuiltinToolDef
+
+models = [
+    AIModel(
+        id='openai:gpt-5',
+        name='GPT 5',
+        builtin_tools=['web_search', 'code_execution'],
+    ),
+    AIModel(
+        id='anthropic:claude-sonnet-4-5',
+        name='Claude Sonnet 4.5',
+        builtin_tools=['web_search'],
+    ),
+]
+
+builtin_tool_definitions = [
+    BuiltinToolDef(
+        id='web_search',
+        name='Web Search',
+        tool=WebSearchTool(),
+    ),
+    BuiltinToolDef(
+        id='code_execution',
+        name='Code Execution',
+        tool=CodeExecutionTool(),
+    ),
+]
+```
+
+If an `agent_options.py` file exists in your current directory, it will be automatically loaded when you run `clai --web`. You can also specify a custom config path with `--config`.
+
+You can also launch the web UI directly from an `Agent` instance using [`Agent.to_web()`][pydantic_ai.Agent.to_web]:
```

Comment:
I think we should have a new `ui/web.md` that explains the feature and the 2 methods of using it (CLI, method). The page should have a screenshot :) And then these CLI docs should link to that doc for more context (after the basic CLI docs above)